Formula/minimal-racket.rb

```diff
@@ -55,13 +59,20 @@ def install
       system "make"
       system "make", "install"
     end
+
+    inreplace racket_config prefix, opt_prefix
```

Comment:
It's possible I'm missing something here, but should this be...
```suggestion
    inreplace racket_config, prefix, opt_prefix
```

Reply:
It should, yes 😄